### PR TITLE
Fix and update names for AutoExposureTime

### PR DESCRIPTION
**Pull request by samuelpowell**
Some cameras name this node `AutoExposureTimeLowerLimit`, other `AutoExposureExposureTimeLowerLimit`. Test at init, catch, and cache.

### CODE REVIEW

**Comment by samuelpowell** [comment]
@ianshmean does this break your camera?

**Comment by samuelpowell** [comment]
Ooops need to merge master

File: src/Camera.jl [hunk 1]
```diff
@@ -28,19 +28,36 @@ export serial, model, vendor, isrunning, start!, stop!, getimage, getimage!, sav
...
     _chunkselect(cam, ["Timestamp"], "timestamps")
+
+    # Discover ambiguous names
+    cam.names["AutoExposureTimeLowerLimit"] = "AutoExposureTimeLowerLimit"
+    cam.names["AutoExposureTimeUpperLimit"] = "AutoExposureTimeUpperLimit"
+    
+    try
+      Spinnaker.get(Spinnaker.SpinIntegerNode(cam, "AutoExposureTimeLowerLimit"))
+    catch
+      @info "renameing exposure"
```
Comment (ianshmean) [inline src/Camera.jl]: I'd go for something a bit clearer:
`@info "Renaming AutoExposureTimeLowerLimit & AutoExposureTimeUpperLimit"`

File: src/Camera.jl [hunk 2]
```diff
@@ -28,19 +28,36 @@ export serial, model, vendor, isrunning, start!, stop!, getimage, getimage!, sav
...
     _chunkselect(cam, ["FrameID", "FrameCounter"], "frame indentification")
     _chunkselect(cam, ["ExposureTime"], "exposure time")
     _chunkselect(cam, ["Timestamp"], "timestamps")
+
+    # Discover ambiguous names
+    cam.names["AutoExposureTimeLowerLimit"] = "AutoExposureTimeLowerLimit"
+    cam.names["AutoExposureTimeUpperLimit"] = "AutoExposureTimeUpperLimit"
+    
+    try
+      Spinnaker.get(Spinnaker.SpinIntegerNode(cam, "AutoExposureTimeLowerLimit"))
```
Comment (ianshmean) [inline src/Camera.jl]: Should be a SpinFloatNode

**Comment by ianshmean** [comment]
Needs to be a SpinFloatNode, as commented. and the message could be clearer

**Comment by samuelpowell** [comment]
Good catch - fixed the integer/float problem. The info was just for debug and removed. Does this still work on your camera?

**Comment by ianshmean** [comment]
Works for me!

**Pull request merged**